cc/base/logging.cc

```diff
@@ -22,7 +22,40 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#ifdef _WIN32
+// START PATCH
+// HACK: This is not a solution just an attempt at see what is going on and getting PyDP to build on windows.
```

Comment:
This comment looks like it's probably out of date. Can you add a comment that says a little more about what this block does?